torch/_inductor/compile_fx.py

```diff
@@ -183,6 +185,8 @@ def compile_fx_inner(
             if aot_mode:
                 return compiled_fn
 
+    output = list(gm.graph.nodes)[-1]
+    assert len(output.args) == 1
```

Comment:
?

Reply:
I was a bit surprised that the "output" node has a single arg with tuple inputs, instead of multiple args. I added an assert to verify that. will add comment